hedera-node/hedera-file-service-impl/src/test/java/module-info.java

```diff
@@ -1,11 +1,18 @@
-module com.hedera.node.app.service.file.impl.test {
+open module com.hedera.node.app.service.file.impl.test {
```

Comment:
that file can be deleted. We decided to not use the module system for unit tests.

Reply:
it is not the case